Add compat data for background(-.*)? properties

teoli2003 opened the pull request.

Elchi3 commented on css/background-clip.json:
```diff
@@ -0,0 +1,167 @@
+      },
+      "Chrome": {
+        "support": true,
+        "notes": ["Support it on the prefixed version of the property only; according https://webkit.org/blog/164/background-clip-text/ WebKit does not include text decorations or shadows in the clipping."]
```
- Remove "it". 
- Use an HTML link.

Elchi3 commented on css/background-position-x.json:
```diff
@@ -0,0 +1,104 @@
+        "obsolete": false
+      }
+    },
+    "Two-value syntax (support for offsets from any edit)": {
```
s/edit/edge

Elchi3 commented on css/background-position-y.json:
```diff
@@ -0,0 +1,104 @@
+        "obsolete": false
+      }
+    },
+    "Two-value syntax (support for offsets from any edit)": {
```
s/edit/edge

Elchi3 commented on css/background-position.json:
```diff
@@ -0,0 +1,155 @@
+        "obsolete": false
+      }
+    },
+    "Four-value syntax (support for offsets from any edit)": {
```
s/edit/edge

Elchi3 commented on css/background-position.json:
```diff
@@ -0,0 +1,155 @@
+      },
+      "Chrome for Android": {
+        "support": true,
+        "notes": ["Some version support only experimental gradients prefixed with -webkit"]
```
Gradients? This is not in the original page

Elchi3 commented on css/background-image.json:
```diff
@@ -0,0 +1,369 @@
+      },
+      "Chrome": {
+        "support": "1.0",
+        "notes": ["Some version support only experimental gradients prefixed with -webkit"]
```
"Some versions only support ..." here and a few times below

Elchi3 commented on css/background-repeast.json:
```diff
@@ -0,0 +1,206 @@
+      },
+      "Firefox": {
+        "support": "13.0",
+        "notes": ["Implemented in bug 548375"]
```
Why is this note needed?

Elchi3 commented on css/background-repeast.json:
```diff
@@ -0,0 +1,206 @@
+      },
+      "Firefox": {
+        "support": "49.0",
+        "notes": ["Implemented in bug 548372"]
```
Why is this note needed?

Elchi3 commented on css/background.json:
```diff
@@ -0,0 +1,254 @@
+        "obsolete": false
+      }
+    },
+    "Support for background-size": {
```
just "background-size"

Elchi3 commented on css/background.json:
```diff
@@ -0,0 +1,254 @@
+        "obsolete": false
+      }
+    },
+    "Support for background-origin and background-clip": {
```
Make this two entries, "background-origin" and "background-clip"

teoli2003 commented:
All comments addressed.
I also noticed a wrong filename (fixed) and added some <code> in notes as we settled to use html there.

I rebased the patches and the commits to have a clean history :-)

Travis-ci is happy too.

teoli2003 commented:
Ok Florian and I have settled to use css/properties/xyz.json instead of css/xyz.json. So I will modify this PR to reflect this :-)

teoli2003 commented:
Moved to css/properties and rebased. Ready for review agian.

Elchi3 commented:
Great work \o/
